code/modules/reagents/newchem/patch.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/weapon/reagent_containers/pill/patch
+	name = "chemical patch"
+	desc = "A chemical patch for touch based applications."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bandaid"
+	item_state = "bandaid"
+	possible_transfer_amounts = null
+	volume = 50
+	apply_type = TOUCH
+	apply_method = "apply"
+	apply_method_plural = "applies"
```

Comment:
God no, bruise pack and ointment code is shit.

It's inherited from pill code because it's supposed to be a pill that does TOUCH applications instead of INGEST.